app/models/filters/filter_base.rb

```diff
@@ -542,7 +550,7 @@ def comparison_patterns
     end
 
     def clean_comparison_pattern(pattern)
-      comparison_patterns.values.detect { |m| m == pattern&.to_sym } || default_comparison_pattern
+      comparison_patterns.values.detect { |m| m == pattern&.to_sym } || comparison_pattern.presence || default_comparison_pattern
```

Comment:
Did you decide the attribute default was enough so it doesn't need to be repeated here?

Reply:
I believe this one still needs to be here... but I could be wrong.  For now, no harm.